src/applications/personalization/profile/selectors.js

```diff
@@ -113,9 +113,4 @@ export function selectVAProfilePersonalInformation(state, fieldName) {
 export const selectHideDirectDepositCompAndPen = state =>
   toggleValues(state)?.[FEATURE_FLAG_NAMES.profileHideDirectDepositCompAndPen];
 
-export const selectShowPaymentsNotificationSetting = state =>
-  toggleValues(state)?.[
-    FEATURE_FLAG_NAMES.profileShowPaymentsNotificationSetting
-  ];
-
```

Comment:
decided to remove this toggle selector and use the Toggler component/hook instead